docs/CHANGELOG.md

```diff
@@ -9,44 +9,64 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <!-- unreleased -->
 ## [Unreleased]
 
-- New optional external logger added
-    ### AUDIT_LOGGER_FORWARD_TO_URL
+### Added
 
-    - Default: _none_
+#### New optional external logger added
+##### AUDIT_LOGGER_FORWARD_TO_URL
```

Comment:
I like the change to smaller headings, but I think the indentation was adding value. It kept everything grouped under a single bullet to match our existing format, and was easier to see the scope visually.

Reply:
Or could it make sense to move this over to docs.chain.link (and new toml) and just link to it from here?

Reply:
Generally what I have done before in changelogs, if the change is large with lots of detail, I put it as a subheading. Small 1-2 liners are a bullet point.

Feels silly to indent 100s of lines of text

Reply:
Why is it silly? The indentation served a purpose both in the structure of the markdown code and the rendered html.

Reply:
Just looks weird

Reply:
What about a collapsible `<details><summary>` block?